### PR TITLE
W-10114645 - Donation History dataset

### PR DESCRIPTION
**Pull request by JoseRossiSF**
# Work Item
[W-10114645](https://gus.lightning.force.com/a07EE00000LKpruYAD)

# Changes
Adding records to dev_org test_data.sql dataset in order to include over seventy Opportunities related to Hard Credits, Soft Credits, and Partial Soft Credits for the Contact "Alice Patterson".

The goal was to create a dataset that can be used by Dev and QA in the development of Donation History Table community feature.

### CODE REVIEW

**Comment by force2b** [comment]
@JoseRossiSF I'm curious - is there a reason you didn't create a Snowfakery recipe to generate the test data? Plus, if this data is very specific to Donation History testing, it should be a new recipe vs. modifying one that could be used by other teams.

**Comment by JoseRossiSF** [comment]
> @JoseRossiSF I'm curious - is there a reason you didn't create a Snowfakery recipe to generate the test data? Plus, if this data is very specific to Donation History testing, it should be a new recipe vs. modifying one that could be used by other teams.

@force2b Manually creating the records seemed like a more straightforward way to accomplish our goal, taking in consideration the records we required were quite specific and not too numerous.
Regarding why I am modifying an existing Dataset instead of creating a new one, that was originally my first approach, but was advised that, given there's plenty of existing datasets, it would be better to simply add to an existing one the records which we need. If you believe a different Dataset should be created instead, let me know and I will make the required changes.

**Review by dcaballeroSf — APPROVED** [review]
LGTM

**Review by mcasco — APPROVED** [review]
LGTM

File: trigger_status.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Opportunity:CRLP_Rollup_TDTM: true
```
Comment (force2b) [inline trigger_status.yml:1]: This file needs to be removed from the PR

**Review by force2b — CHANGES_REQUESTED** [review]
One change needed, but overall I do feel that the better solution here would have been to create a new snowfakery script that built data exactly as your product needs it. The SQL file modified here is pretty old and has never been updated or brought up to date with any of the more recent changes to NPSP.

File: trigger_status.yml (hunk 1 above)
Comment (JoseRossiSF) [reply trigger_status.yml]: Agreed, thanks for pointing it out

**Review by force2b — APPROVED** [review]
Overall this will probably work ... at least for this year. Because the dates are hardcoded into the SQL file the data is very dated. This is another primary advantage of Snowfakery. The dates be automatically created as relative to the current date.

**Review by psadie-sfdo — APPROVED** [review]

**Pull request merged**